### ISSUE TITLE
Reset Password Link Works More Than Once

### ISSUE THREAD

**Issue by giancorderoortiz** [SAP/cloud-commerce-spartacus-storefront#3644]
**Setup:**
1. Configure backend to send emails. 

**Steps to Reproduce:** 
1. Navigate to spartacus storefront
2. Register a new user (i.e. john.doe@acme.com). 
3. Add items to cart and checkout. 
4. Signout. 
5. Click on sign-in button and then on Forgot Password? link. 
6. Enter email (john.doe@acme.com)
7. Wait a few minutes and check mail inbox. 
8. Open email received from "Customer Services Team", titled "Forgot password". 
9. Click on "update-password link"
10. Follow steps and update password. 
11. Wait 5 -10 minutes. 
12. Repeat steps 8 - 10. 

**Expected Results:**
12. Step must fail. Password reset link should work only once. 

**Actual Results:**
12. Step is successful. Able to update password many times within the time frame given (i.e. 30 seconds)

**Comment by hackergil** [comment]
Backend should take care of this. This is out of scope for Spartacus.
